README.md

```diff
@@ -34,16 +33,40 @@ PYTHONPATH=$PYTHONPATH:$EBROOTREFRAME:$eessihome reframe \
     -r --performance-report
 ```
 
-## Improvements in PR #11 (2022-12-14)
-
-- features to filter out CUDA modules in non-GPU nodes and non-CUDA modules in GPU nodes
-    - requires adding `features` `cpu` and/or `gpu` to the partitions in the site config file
-- support for specifying modules
-    - via `--setvar modules=<modulename>`
-- support for specifying systems:partitions
-    - via `--setvar valid_systems=<comma-separated-list>`
-- support for overriding tasks, cpus
-    - via `--setvar num_tasks_per_node=<x>` and/or `--setvar num_cpus_per_task=<y>`
-- support for setting additional environment variables
-    - via `--setvar variables=<envar>:<value>`
+## Configuring GPU/non-GPU partitions in your site config file:
+
+- running GPU jobs in GPU nodes
+    - add feature `gpu` to the GPU partitions
+
+- running non-GPU jobs in non-GPU nodes
+    - add feature `cpu` to the non-GPU partitions
+
+- running GPU jobs and non-GPU jobs on gpu nodes
+    - add both features `cpu` and `gpu` to the GPU partitions
+    ```
+    'features': ['cpu', 'gpu'],
+    ```
+
+- setting the number of GPUS per node for a partition:
+    ```
+    'access': ['-p <partition_name> --gpus-per-node=<x>'],
```

Comment:
We should set this in the test itself, we can use the number of GPU devices as default (and make it overwriteable by `--set-var`)